chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetProps.ts

```diff
@@ -56,6 +57,7 @@ export interface ILiveChatWidgetProps {
     startChatErrorPaneProps?: IStartChatErrorPaneProps;
     styleProps?: ILiveChatWidgetStyleProps;
     telemetryConfig: ITelemetryConfig;
+    notificationPaneProps?: INotificationPaneProps;
```

Comment:
We need to update our dev guide for all the schema changes

Reply:
Have a separate task to update documentation in a later PR -> [Task 3426607](https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3426607): Create md for Notification Banner